docs/console/console-extensibility/sso.md

```diff
@@ -1,86 +1,164 @@
+
+The Console endpoint that handles extension authorization is the following one, preceded by the Console base URL:
+
+`/tenants/{tenantId}/extension/{extensionId}/authz?state={state}`
+
+Here an example:
+
+```js
+function performExtensionLogin() {
+   const tenantId = 'acme'
+   const extensionId = 'acme-extension-123'
    
-     ```json
-     {
-       "code": "CODE",
-       "state": "STATE"
-     }
-     ```
-
-5. **Receive Access Token**
-   - The Console will respond with an access token. The extension's backend returns this token to its frontend, where it can be used to access protected resources in the backend.
-
-6. **Validate the Access Token**
-   - The extension backend can validate the token by calling the following Console endpoint:
+   // Remember to store the state and validate it throughout the login flow
+   const state = 'foo'
    
-     ```
-     GET CONSOLE_BASE_URL/jwks
-     ```
+   window.location.replace(`https://console.cloud.mia-platform.eu/tenants/${tenantId}/extension/${extensionId}/authz?state=${state}`)
+}
+```
+
+```html
+<!-- When user clicks the button the login flow is started -->
+<button onclick="performExtensionLogin()">Login with Mia-Platform</button>
+```
+
+:::info
+How the extension generates and stores the `state` is not scope of this guide.
+:::
+
+## Extension Callback URL
+
+As per [OAuth 2.0 specification][ietf-redirection-endpoint]: *"The redirection endpoint URI MUST be an absolute URI [...] The endpoint URI MAY include an "application/x-www-form-urlencoded" formatted query component which MUST be retained when adding additional query parameters. The endpoint URI MUST NOT include a fragment component"*
+
+:::info
+Currently, only valid *HTTPS* URLs can be configured as redirection endpoints.  
+:::
+
+In OAuth 2.0 with Authorization Code Grant, the *authorization server* redirects the *user-agent* back to the *client* (in this case the extension frontend) using the redirection URI configured earlier (the extension's *Callback URL*), providing an authorization `code` and the `state` provided by the client earlier.
+
+Use both the authorization `code` and the `state` to requests an access token from the Console token endpoint `/api/oauth/token`.
+
+:::info
+For safety reasons, all modern browsers block Cross Origin requests, thus you need to implement a proxy endpoint in order to contact the Console token endpoint.
+:::
+
+## Extension Token Endpoint
+
+This endpoint exposed by the extension backend is needed to exchange the *Authorization Code* received on the extension *Callback URL* with a valid access token.
+
+This endpoint just acts as a proxy towards the following Console token endpoint, preceded by the Console base URL:
+
+`/api/oauth/token`
```

Comment:
I'd share an example of the request body as well, moreover there is no information about the HTTP method.

Please be more explicit on all API interfaces